multicore_runtime/static_files.py

```diff
@@ -14,18 +14,23 @@
 #
 """Static file serving functionality invoked by wsgi_config.py."""
 
+import datetime
 import httplib
 import logging
 import mimetypes
 import os
 import re
 
+from werkzeug import datastructures
+from werkzeug import http
 from werkzeug import wrappers
 from werkzeug import wsgi
 
 
 def static_app_for_regex_and_files(url_re, files_mapping, upload_re,
-                                   mime_type=None):
+                                   mime_type=None,
+                                   http_headers=None,
+                                   expiration=None):
```

Comment:
I wrote this comment because the docstring says expiration is a string; however, looking at the code, it looks like expiration is treated as an integer. Maybe the docstring is incorrect?